applications/precompiled_pva/cpp/pva_unsharp_mask/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I don't see an `add_subdirectory` for this file, is this an embedded standalone project? Could you provide a small README for getting started?

It looks like this contains multiple implementations of the unsharp mask.  Could you maybe provide details on the file contents of this project and how to compare the two approaches? Are they directly comparable?

Would be nice to integrate with the application project via `ExternalProject` but not critical.

Reply:
Removed the CUPVA based algorithm source code and related documentation for now.
It will be added in the upcoming releases.